Dockerfile

```diff
@@ -128,5 +131,8 @@ COPY --from=build-app /app/build-bff /app/build-bff
 COPY --from=build-app /app/node_modules /app/node_modules
 COPY --from=build-app /app/package.json /app/package.json
 
+RUN apt-get update \
+  && apt-get install nano
```

Comment:
Voor mijn beeld, waarom installeren we nano?

Reply:
Alleen op de ACC server. Dat is handig als ik vanaf de server beetje moet debuggen e.d